unison-src/transcripts/move-namespace.output.md

```diff
@@ -278,7 +278,7 @@ I should be able to move the root into a sub-namespace
   
   
   
-  □ 1. #hu6p22qbe4 (start of history)
+  □ 1. #j17hvtt1rm (start of history)
```

Comment:
What might have caused this change? Did some of the builtins change hashes or something?

Reply:
I tried to keep these transcript changes cleanly separated by commit. Looks like a change in `unison-src/transcripts-using-base/base.u`.

https://github.com/unisonweb/unison/pull/3737/commits/bfe8364bb0f4b125e86f62993e1d12d8da3040a1